img/s/swinburne.edu.au.svg

```diff
@@ -1 +1 @@
.3v-.1h-.5v1h.2V6h.3zM15 7.3v.2h.3v.8h.2v-.9h.2zm1.5 1a.3.3 0 0 1-.1 0h-.3v-.5h.3-.3v-.4h.3-.5v.9h.6m.7 0a.5.5 0 0 1-.4-.2.4.4 0 0 1 0-.3v-.3h.4a.4.4 0 0 1 .2 0v-.1a.6.6 0 0 0-.2 0 .6.6 0 0 0-.5 0 .5.5 0 0 0-.1.4l.1.4.4.2h.3v-.2a.5.5 0 0 1-.2 0m1.2.1v-.8a.2.2 0 0 1 0-.1h-.2v.3H18v-.3h-.2v1h.2v-.6h.5v.5h.1m1-1a.4.4 0 0 1 0 .2V8l-.7-.8h-.1v1.1h.1v-.9l.8.9v-1zm.8.1c.2 0 .4.2.4.4a.4.4 0 0 1-.1.3.4.4 0 0 1-.3.2c-.2 0-.4-.2-.4-.5 0-.2.2-.4.4-.4zm0 0a.5.5 0 0 0-.3 0l-.2.4a.5.5 0 0 0 .5.6l.4-.2a.5.5 0 0 0 .1-.4.5.5 0 0 0-.5-.5m1.3 1H21v-1a.1.1 0 0 1 0 .1v1h.6v-.2m.6-.8c.2 0 .4.2.4.4a.4.4 0 0 1-.2.3.4.4 0 0 1-.2.1c-.3 0-.4-.1-.4-.4 0-.2.1-.4.4-.4zm0-.1a.5.5 0 0 0-.6.5.5.5 0 0 0 .5.5.5.5 0 0 0 .4-.1.5.5 0 0 0 .2-.4.5.5 0 0 0-.5-.5m1.4.5v.4a.4.4 0 0 1-.2 0 .4.4 0 0 1-.3 0 .4.4 0 0 1-.2-.4.4.4 0 0 1 .2-.3.4.4 0 0 1 .3 0h.2v-.1a.7.7 0 0 0-.2 0c-.4 0-.6.1-.6.4 0 .2 0 .3.2.4l.4.2.3-.1v-.4h-.1zm.8-.5-.2.4-.2-.3h-.2l.3.4v.5h.2v-.5l.3-.4h-.2z"/><path d="M0 0h13.2v13.2H0z"/><path fill="#fff" d="M2.1 4.4h-.7V4c.2.2.5.2.8.2.5 0 .8-.2.8-.6 0-.4-.2-.5-.5-.7L2 2.6c-.3-.2-.5-.4-.5-.8 0-.5.5-.9 1-.8h.8v.4c-.2-.2-.4-.2-.7-.2-.4 0-.8.2-.7.6 0 .2.1.3.4.5l.5.3c.4.2.7.3.8 1 0 .7-.7.9-1.4.8m5.5-3-1 3-1-2.2-.8 2.3-1.1-3.1a.3.3 0 0 0 0-.1.4.4 0 0 0-.2-.1h.6v.2L5 3.6l.6-1.8-.1-.4-.1-.1a1 1 0 0 0-.1-.1h.6a.3.3 0 0 0 0 .2l.8 2.2.7-2.2v-.1a.2.2 0 0 0 0-.1h.5a1.1 1.1 0 0 0-.2.1m.5.8h-.4a.3.3 0 0 1 .1.1.3.3 0 0 1 0 .1v2.1H8V2zm3.5-.8v3.2L9 1.6V4a.8.8 0 0 0 0 .2v.2h-.3V1.3a.2.2 0 0 0 0-.1H9l2.2 2.5V1.2h.4a.8.8 0 0 0 0 .1M3.4 6.5h-.3V8a.8.8 0 0 0 .3 0c.5 0 .9-.2.9-.7 0-.5-.5-.7-.9-.7m0-1.3h-.3v1.1h.3c.5 0 .7-.2.7-.6 0-.3-.3-.5-.7-.5m.2 3h-.9v-3a.4.4 0 0 0 0-.2.7.7 0 0 0 0-.1h1c.4 0 .7.3.7.7 0 .3-.1.5-.3.6l-.2.1h.2c.3.2.5.4.5.9s-.4.9-1 .9m4-3.1v2c0 .7-.5 1.1-1.2 1.1s-1.3-.4-1.3-1v-2a.7.7 0 0 0 0-.2h.4v2c0 .6.4 1 .9 1a.8.8 0 0 0 .8-.9v-2a.8.8 0 0 0 0-.1h.4a.8.8 0 0 0 0 .1m1.4 0a.7.7 0 0 0-.3.1v1.1a1.9 1.9 0 0 0 .3 0c.4 0 .7-.2.7-.6 0-.3-.1-.5-.7-.6m1.1 3L9 6.6v-.1a.2.2 0 0 0-.1 0v1.7h-.4V5.4a1.5 1.5 0 0 0 0-.3V5h1c.4 0 .7.2.7.7 0 .5-.4.8-.8.8l.2.3.8 1 .3.3h.2zM7 9v3L4.3 9.3v2.4a.8.8 0 0 0 0 .2v.2H4l.1-.1V8.9a.2.2 0 0 0-.1-.2h.5l2.2 2.6V8.7H7A.8.8 0 0 0 7 9m2.6 3h-2V9a.3.3 0 0 0 0-.2h1.6v.4a.3.3 0 0 0-.1-.1H8v1h1.2v.4a.8.8 0 0 0-.1-.1H8v1.4h1a1 1 0 0 0 .4 0 .5.5 0 0 0 .2-.1zM1.9 9.8a4.8 4.8 0 0 1 .2 0 .5.5 0 0 1 .1.1v.2h.1v-.2h.2v-.1.2a.1.1 0 0 1 0 .1l-.1.1a.3.3 0 0 1 .2 0l.1.1v.2a.3.3 0 0 1-.1 0h-.2v-.2h-.1v.2l.1.1a.2.2 0 0 1 0 .1l-.2.1c0 .1 0 0 0 0a.1.1 0 0 1-.1-.2.4.4 0 0 1 0-.1v-.1a.8.8 0 0 1 0 .1.1.1 0 0 1-.1 0 .8.8 0 0 1-.1 0 .2.2 0 0 1-.1 0v-.2a.2.2 0 0 1 .2 0h.1a.7.7 0 0 0 0-.1H2l-.1-.2v-.1c0-.1 0 0 0 0v-.1m9.4 0a5.7 5.7 0 0 0-.1 0 1 1 0 0 0-.1 0 .4.4 0 0 0 0 .1v.1l-.1.1v-.2h-.2l-.1-.1v.2a.1.1 0 0 0 0 .1h.2v.1h-.1a.3.3 0 0 0-.1 0 .4.4 0 0 0-.1 0l-.1.2a.1.1 0 0 0 .1 0v.1h.2v-.2h.2v.1h-.1v.2a.2.2 0 0 0 0 .1h.1c.1 0 0 0 0 0h.1a.1.1 0 0 0 0-.1.4.4 0 0 0 0-.1v-.1a.8.8 0 0 0 .1 0v.1h.1a.2.2 0 0 0 .1 0 .5.5 0 0 1 .1 0v-.2h-.1a.1.1 0 0 0-.1 0h-.1s-.1 0 0 0a.7.7 0 0 1 0-.1l.2-.2v-.2"/></svg>
```

Comment:
The logo is not legible at this level of precision.

Reply:
Updated with new logo